Containerfile

```diff
@@ -149,6 +149,8 @@ CMD ["bundle", "exec", "rails", "server", "-b", "0.0.0.0", "-p", "3000"]
 ## Development
 FROM base AS development
 
+ENV TMPDIR=/var/tmp
```

Comment:
Without this, codespace setup would fail with

```
== initializing the application ==
tmpdir is world-writable: /tmp
system temporary path is world-writable: /tmp
/tmp is world-writable: /tmp
. is world-writable: /app
rails aborted!
argumenterror: could not find a temporary directory
/app/config/initializers/carrierwave.rb:1:in `<main>'
/app/config/environment.rb:5:in `<main>'
tasks: top => app_initializer:setup => environment
(see full trace by running task with --trace)
```

Reply:
Well that's weird. Normally I'd complain fiercely about this because of how `/tmp` is often a `tmpfs` mount on Linux (meaning in RAM), but `/var/tmp` is often not, but since this is within a `Containerfile` and mounts are almost never `tmpfs` at that layer, it's chill.